fronted/src/utils/NewsApi.js

```diff
@@ -4,8 +4,15 @@ const fetchFunction = (url, headers) => {
 
```

Comment:
no need to duplicate `fetchFunction` in 2 files. You need to create it only once and import to the placde where you need it